core/primitives/src/stateless_validation.rs

```diff
@@ -340,6 +340,13 @@ impl ChunkStateWitness {
         }
     }
 
+    /// Returns size in bytes of borsh-serialized state witness.
+    /// NOTE: this is potentially expensive operation and should only be
+    /// used in tests.
+    pub fn borsh_size(&self) -> ChunkStateWitnessSize {
```

Comment:
If we stop caring about passing raw_size to process_chunk_state_witness fn, we can get rid of this function as well.

Reply:
dummy value would where it doesn't matter, but most test cases are actually on orphan pool where it is actually needed
decided to completely get rid of it and just inline usage for now: 535cef90f83588f21b067f039125a8dd46cde626